src/components/AgmExtension/AgmExtensionEvaluation.vue

```diff
@@ -120,7 +110,16 @@ export default class AgmExtensionEvaluation extends Vue {
   @Prop({ required: true }) readonly data!: AgmExtEvalIF
 
   get dueDateString (): string {
-    return (DateUtilities.formatYyyyMmDd(this.data.agmDueDate))
+    const date = (DateUtilities.yyyyMmDdToDate(this.data.agmDueDate))
+    const pacificDate = DateUtilities.dateToPacificDate(date, true)
+    if (pacificDate) return `${pacificDate} at 11:59 pm Pacific time`
+    return ''
+  }
```

Comment:
Paul, I'm looking the UXPin, I don't see the "at x Pacific time".
In your temp URL:
![due date paul](https://github.com/bcgov/business-filings-ui/assets/122301442/129b6f99-a34d-4dbc-9cc9-2bff2a238d38)

I guess we don't need that part?

Also, please Paul if you can add an empty line between the two getters.

Reply:
Fixed, thanks! Resolving.